dev-support/ranger-docker/Dockerfile.ranger-base

```diff
@@ -25,7 +25,7 @@ RUN apt-get update && \
     pip3 install requests
 
 # Set environment variables
-ENV JAVA_HOME      /usr/lib/jvm/java-8-openjdk-amd64
+ENV JAVA_HOME      /usr/lib/jvm/java-8-openjdk-arm64
```

Comment:
RANGER-4181 replaced hardcoding of "amd64" with TARGETARCH. I suggest to consider a similar approach here.

Reply:
Sure, I've updated the patch.